code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -714,6 +714,6 @@
 	//if you made it this far: congratulations! you are now a religious zealot!
 	target.mind.make_zealot(missionary, convert_duration, team_color)
 
-	target << sound('sound/misc/wololo.ogg', 0, 1, 25)
+	SEND_SOUND(target, sound('sound/misc/wololo.ogg', volume = 25))
 	missionary.say("WOLOLO!")
-	missionary << sound('sound/misc/wololo.ogg', 0, 1, 25)
+	SEND_SOUND(missionary, sound('sound/misc/wololo.ogg', volume = 25))
```

Comment:
these ones seemed like a bug to me, it's trying to get to channel 25, which doesn't make sense, but lowering the volume to 25% does make sense